src/scvi/data/_download.py

```diff
@@ -1,57 +1,19 @@
 import logging
 import os
-import urllib
 from typing import Optional
 
-import numpy as np
-
-from scvi.utils import track
+import scanpy as sc
```

Comment:
We have no requirement on scanpy in scvi-tools and would like to keep it (the one that have are optional imports). You are using a private (_XXX) function from scanpy, see the Zulip discussion from last week why this is a bad idea.

Reply:
Yes of course. This will be removed once we figure out what works and what doesn't